### PR TITLE
Docker image for kerberized CDH 5

### PR DESCRIPTION
**Pull request by ilfrin**
Based on the prestodb/hdp2.5-hive-kerberized image, with some changes seen at teradatalabs/cdh5-hive-kerberized

Tested manually (create table in hive, insert, select)

### CODE REVIEW

File: prestodb/cdh5-hive-kerberized/files/var/kerberos/krb5kdc/kadm5.acl [hunk 1]
```diff
@@ -0,0 +1 @@
+*/admin@LABS.TERADATA.COM	*
```
Comment (losipiuk) [inline prestodb/cdh5-hive-kerberized/files/var/kerberos/krb5kdc/kadm5.acl]: newline

Comment (ilfrin) [reply prestodb/cdh5-hive-kerberized/files/var/kerberos/krb5kdc/kadm5.acl]: done

File: prestodb/cdh5-hive-kerberized/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,82 @@
+# Copyright 2016 Teradata
```
Comment (kokosing) [inline prestodb/cdh5-hive-kerberized/Dockerfile]: Please update copyright

File: prestodb/cdh5-hive-kerberized/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+RUN /usr/sbin/kadmin.local -q "addprinc -randkey presto-server/presto-worker-2.docker.cluster@LABS.TERADATA.COM"
+RUN /usr/sbin/kadmin.local -q "addprinc -randkey HTTP/presto-master.docker.cluster@LABS.TERADATA.COM"
+RUN /usr/sbin/kadmin.local -q "addprinc -randkey presto-client/presto-master.docker.cluster@LABS.TERADATA.COM"
+RUN /usr/sbin/kadmin.local -q "addprinc -randkey hive/presto-master.docker.cluster@LABS.TERADATA.COM"
+RUN mkdir -p /etc/presto/conf
+RUN /usr/sbin/kadmin.local -q "xst -norandkey -k /etc/presto/conf/presto-server.keytab presto-server/presto-master.docker.cluster presto-server/presto-worker.docker.cluster presto-server/presto-worker-1.docker.cluster presto-server/presto-worker-2.docker.cluster"
+RUN /usr/sbin/kadmin.local -q "xst -norandkey -k /etc/presto/conf/presto-server-HTTP.keytab HTTP/presto-master.docker.cluster"
+RUN /usr/sbin/kadmin.local -q "xst -norandkey -k /etc/presto/conf/presto-client.keytab presto-client/presto-master.docker.cluster"
+RUN /usr/sbin/kadmin.local -q "xst -norandkey -k /etc/presto/conf/hive-presto-master.keytab hive/presto-master.docker.cluster"
+RUN chmod 644 /etc/presto/conf/*.keytab
```
Comment (kokosing) [inline prestodb/cdh5-hive-kerberized/Dockerfile]: [optional]

do we need to have so many layers?

maybe RUN `cmd && cmd2` would be better?

File: prestodb/cdh5-hive-kerberized/README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# cdh5-hive-kerberized [![][layers-badge]][layers-link] [![][version-badge]][dockerhub-link]
+           
+[layers-badge]: https://images.microbadger.com/badges/image/prestodb/cdh5-hive-kerberized.svg
+[layers-link]: https://microbadger.com/images/prestodb/cdh5-hive-kerberized
+[version-badge]: https://images.microbadger.com/badges/version/prestodb/cdh5-hive-kerberized.svg
+[dockerhub-link]: https://hub.docker.com/r/prestodb/cdh5-hive-kerberized
```
Comment (kokosing) [inline prestodb/cdh5-hive-kerberized/README.md]: do these links work?

**Pull request merged**